src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,32 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	gentity_t *tmp = g_entities;
+	int miner = 0;
+	for (int i = 0; i < level.num_entities; i++, tmp++) 
```

Comment:
Can rewrite this as:

ForEntities<MiningComponent>([](Entity& entity, MiningComponent& mining) {});

Reply:
This is an optional change though